tests/sclT0009-unused-symbols/sym01.exec

```diff
@@ -0,0 +1,2 @@
+# format: exec
+exec:
```

Comment:
I have asked a question here: https://github.com/sn127/utils/issues/16

Otherwise, what you describe sounds good.